client/verta/tests/unit_tests/registry/test_model_dependencies.py

```diff
@@ -126,5 +126,26 @@ def test_class_module_names(dependency_testing_model) -> None:
         'verta',
         'yaml',
     }
-    extracted_modules: Set[str] =  md.class_module_names(dependency_testing_model)
+    extracted_modules: Set[str] = md.class_module_names(dependency_testing_model)
     assert set(extracted_modules) == set(expected_modules)
+
+
+def test_package_names(dependency_testing_model) -> None:
+    """Verify that distribution package names are returned correctly for the
+    given module names when they differ from the package name.  Test fixture
+    ensures packages required for the test are installed or this test will be
+    skipped.
+    """
+    expected_packages = {
+        'sklearn': ['scikit-learn'],
+        'PIL': ['Pillow', ],
```

Comment:
Maybe black would catch this in a future PR, but

```suggestion
        'PIL': ['Pillow'],
```

Reply:
fixed